ui/webpack.common.ts

```diff
@@ -55,6 +69,7 @@ module.exports = {
       template: './assets/index.html',
       favicon: './assets/images/favicon.ico',
       inject: 'body',
+      base: BASE_PATH.slice(0, -1),
```

Comment:
if `BASE_PATH` doesn't end with a slash, this is gonna be wonky. what do you think of adding a comment that points to the external file that makes sure basepath is properly terminated so folks reading this know we got it covered?

Reply:
it should be all cleaned in the `ui/src/utils/env.ts` file, so you can throw any gibber jabber in there and it'll make it look like a folder to webpack

Reply:
>it should be all cleaned in the ui/src/utils/env.ts file, so you can throw any gibber jabber in there and it'll make it look like a folder to webpack

word!

i was poorly trying to say it might be reader friendly to add a comment saying as much (provided it specifically includes the phrase jibber jabber), just to prevent people from having to trace this import to its source to see it's handled there.